### PR TITLE
Get connection status from wg interface instead of pinger

### PR DESCRIPTION
**Pull request by alphadose**

### CODE REVIEW

File: nmproxy/metrics/metrics_helper.go [hunk 1]
```diff
@@ -1,36 +1,24 @@
...
-			pingStats := pinger.Statistics()
-			if pingStats.PacketsRecv > 0 {
-				connected = true
-				return
-			}
+		return false
+	}
+	for _, peer := range ifacePeers {
+		if peer.PublicKey.String() == peerPublicKey {
+			return peer.LastHandshakeTime.After(time.Now().Add(-3*time.Minute)) && peer.ReceiveBytes+peer.TransmitBytes > 0
```
Comment (abhishek9686) [inline nmproxy/metrics/metrics_helper.go:20]: `time.Now().Sub()` method can be used

Comment (alphadose) [reply nmproxy/metrics/metrics_helper.go:20]: time.Sub() returns time.Duration which additionally needs to be casted to time.Time

but time.Add returns time.Time

**Review by theguy951357 — CHANGES_REQUESTED** [review]
I am getting weird issues on this one. when I join two nodes, it gets rid of the firs one that joined.




journalctl logs:

an 31 16:57:53 fedoraQA netclient[91722]: [netclient] 2023-01-31 16:57:53 network: network  received delete request for %s c9bafa27-190e-4e75-a37c-206d9a25904a

Jan 31 16:57:53 fedoraQA netclient[91722]: [netclient] 2023-01-31 16:57:53 removing wireguard config

Jan 31 16:57:53 fedoraQA netclient[91722]: [netclient] 2023-01-31 16:57:53 failed to leave, please check that local files for network network were removed

Jan 31 17:00:09 fedoraQA netclient[91722]: [netclient] 2023-01-31 17:00:09 error updating wireguard peersopen /etc/netclient/netmaker.conf: no such file or directory




then when I rejoin that node, the other one gets knocked off.

**Review by theguy951357 — APPROVED** [review]
that error seems to be unrelated to this.
journalctl -fu netclient shows that pinger is no longer being used
pass.

**Comment by 0xdcarns** [comment]
resolve conflicts, and merge will commence @alphadose

**Comment by alphadose** [comment]
@0xdcarns done

**Pull request merged**